Fix traceback from using wrong name.

Patch by Peter `MathFox' Roozemaal.

http://bugs.gentoo.org/97762
http://sourceforge.net/tracker/index.php?func=detail&aid=801329&group_id=74836&atid=542163

--- qct/specializations.py
+++ qct/specializations.py
@@ -131,9 +131,9 @@
         mods = self.getModsFor('incomeMod', employer)
         if resident:
             if resident.job == trainer:
-                return prospect.level * INCOMEMULT * mods
+                return prospect.level * constants.INCOMEMULT * mods
             else:
-                return prospect.level * INCOMEMULT / 2 *  mods
+                return prospect.level * constants.INCOMEMULT / 2 *  mods
         else:
             return 0
 
